### PR TITLE
Remove ES6 features from index.ejs

### PR DESCRIPTION
**Pull request by hjheath**

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-182-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://182-pr-onfido-sdk-ui-onfido.surge.sh

File: src/index.ejs [hunk 1]
```diff
@@ -56,8 +56,8 @@
 
       var createSDK = function(token) {
         window.onfidoOut = Onfido.init({
-          token,
-          useModal,
+          token: token,
```
Comment (stefaniacardenas) [inline src/index.ejs]: Cool, there is an implicit unpacking on line 95 that might break

Comment (stefaniacardenas) [inline src/index.ejs]: Yes, it does break. Just tested it on IE11

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-182-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://182-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**